js/renderer.js

```diff
@@ -29,29 +29,35 @@ class Renderer {
 
         for (layer of bg.layers) {
             const img = document.getElementById(layer.image);
-            this.fillCanvasWithImage(img, layer.x, layer.y);
+            const zoomFactor = (this.camera.zoom) / (Math.pow(this.camera.zoom, 1 - layer.relativeSpeed));
```

Comment:
```suggestion
            const zoomFactor = Math.pow(this.camera.zoom, layer.relativeSpeed);
```
Tested this simplified calculation and it seems to work.